pydrive2/files.py

```diff
@@ -27,6 +28,13 @@ class FileNotUploadedError(RuntimeError):
 class ApiRequestError(IOError):
     """Error while making any API requests."""
 
+    def __init__(self, http_error):
```

Comment:
I think we should still initialize it in a regular way - it should support a regular `ctr` with cause and call `super`

Reply:
@shcheklein have updated to `chain` it `from HttpError` (so __cause__ is set)

Reply:
I don't see it, super is called w/o cause